OrangeIguanas/Bangazon-cli
# Can add payment method to active customer

# Can add payment method to active customer

# Related Ticket(s)

## Testing

[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

## Documentation

[x] There is new documentation in this pull request that must be reviewed..

## Deploy Notes

```sh
git pull --prune
git checkout ps_payments
```

## Steps to Test
Run cli.py, press '2' to set a customer to 'active' status, then press '3' to add a payment type and check your DB to ensure that the foreign key matches that of the active customer. 

1.
